### PR TITLE
Adding import plugin and refactor

### PR DESCRIPTION
**Pull request by raparicio6**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -74,12 +82,12 @@ module.exports = {
     "array-callback-return": "error",
     "block-scoped-var": "error",
     "class-methods-use-this": "error",
-    complexity: ["error", 10],
+    "complexity": ["error", 10],
```
Comment (damfinkel) [inline index.js]: keys that don't need `""` shouldn't have them

File: index.js [hunk 2]
```diff
@@ -15,14 +15,22 @@ module.exports = {
...
+  plugins: ["prettier", "import"],
   globals: {
     __DEV__: true
   },
   rules: {
     // Prettier
     "prettier/prettier": ["error", { printWidth: 110, singleQuote: true }],
 
+    // Import
+    "import/order": ["error", { groups: [["builtin", "external", "internal"]] }],
```
Comment (damfinkel) [inline index.js]: in react we have a different rule, but I think it's fine to have this one as the general one

File: index.js [hunk 3]
```diff
@@ -158,17 +166,17 @@ module.exports = {
...
     "keyword-spacing": "error",
     "line-comment-position": "error",
     "linebreak-style": "error",
     "lines-between-class-members": "error",
     "max-depth": "error",
-    "max-lines": "error", // NOTE: Disable Node
+    "max-lines": "error",
     "max-nested-callbacks": ["error", { max: 3 }],
-    "max-params": ["error", { max: 4 }],
+    "max-params": ["error", 4],
```
Comment (damfinkel) [inline index.js]: Although in the example in docs it's shown like this, the doc says it should be used as { "max": 4 }. Are you sure this works? Link to docs: https://eslint.org/docs/rules/max-params

File: index.js (hunk 1 above)
Comment (raparicio6) [reply index.js]: I put quotes to the keys that weren't with quotes in order to make all rules with uniform and pretty syntax

File: index.js (hunk 3 above)
Comment (raparicio6) [reply index.js]: Yes I am. I tested that rule and works ok. In Node we use that approach so I moved it here to maintain all rules syntax updated.

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-wolox",
-  "version": "2.3.0",
+  "version": "3.0.0",
```
Comment (mcallegari10) [inline package.json]: 2.4 it's more suitable IMO

**Pull request merged**